nav2_behavior_tree/include/nav2_behavior_tree/plugins/condition/globally_updated_goal_condition.hpp

```diff
@@ -57,7 +57,10 @@ class GloballyUpdatedGoalCondition : public BT::ConditionNode
    */
   static BT::PortsList providedPorts()
   {
-    return {};
+    return {
```

Comment:
For all instances: the BT xml index needs to be updated: https://github.com/ros-planning/navigation2/blob/main/nav2_behavior_tree/nav2_tree_nodes.xml

Reply:
And description perhaps?

Reply:
I don't even know what the description should be...

Reply:
Remember that that file should be generated automatically using this function, not edited by hand...

https://github.com/BehaviorTree/BehaviorTree.CPP/blob/master/include/behaviortree_cpp/xml_parsing.h#L60

let me fix that

Reply:
The description is that this is the navigation goal or navigation goals (if using navigate through poses).

I'm not terribly concerned how you update the XML index, just that its in sync

Reply:
More about syncing the XML in #4073

Reply:
Still need the XML index updates for this PR for these items